browser/resources/settings/email_aliases_page/content/email_aliases_signin_page.tsx

```diff
@@ -74,18 +73,24 @@ const BeforeSendingEmailForm = ({ suggestedAuthEmail, emailAliasesService }:
   </SpacedCol>
 }
 
-const AfterSendingEmailMessage = ({ authEmail, emailAliasesService }:
-  { authEmail: string, emailAliasesService: EmailAliasesServiceInterface }) => {
-  const onClick = () => {
-    emailAliasesService.cancelAuthenticationOrLogout()
-  }
+const AfterSendingEmailMessage = (
+  { authEmail, emailAliasesService, errorMessage }:
+  { authEmail: string, emailAliasesService: EmailAliasesServiceInterface,
+    errorMessage: string | undefined }) => {
   return <SpacedCol>
     <h4>{formatLocale('emailAliasesLoginEmailOnTheWay', { $1: authEmail })}</h4>
     <div>{getLocale('emailAliasesClickOnSecureLogin')}</div>
     <div>
-      {formatMessage(getLocale('emailAliasesDontSeeEmail'),
-       { tags: { $1: (content) => <a href='#'
-                                     onClick={onClick}>{content}</a> } })}
+      {getLocale('emailAliasesDontSeeEmail')}
+    </div>
+    {errorMessage && <Alert>
+      {errorMessage + ' ' + getLocale('emailAliasesAuthTryAgain')}
```

Comment:
in general, I think we shouldn't be doing any manipulation of localized strings - generally better to have them in the same string with some placeholders/wrappers